pkg/cri/resource-manager/resource-manager.go

```diff
@@ -289,6 +298,45 @@ func (m *resmgr) loadConfig() error {
 	return nil
 }
 
+// setupConfigSignal sets up a signal handler for reloading forced configuration.
+func (m *resmgr) setupConfigSignal(signame string) error {
+	if signame == "" {
+		return nil
+	}
+
+	m.Info("setting up signal %s to reload forced configuration", signame)
+
+	sig := unix.SignalNum(signame)
+	if int(sig) == 0 {
+		return resmgrError("invalid forced configuration reload signal '%s'", signame)
+	}
+
+	m.signals = make(chan os.Signal, 1)
+	signal.Notify(m.signals, sig)
+
+	go func(signals <-chan os.Signal) {
+		for {
+			select {
+			case _, ok := <-signals:
+				if !ok {
+					return
+				}
+			}
+
+			m.Info("reloading forced configuration %s...", opt.ForceConfig)
+
+			m.Lock()
+			if err := pkgcfg.SetConfigFromFile(opt.ForceConfig); err != nil {
+				m.Error("failed to reload forced configuration %s: %v",
+					opt.ForceConfig, err)
```

Comment:
Is this enough? Should we do something similar to what [SetConfig in resource-manager.go](https://github.com/intel/cri-resource-manager/blob/670360625f165da227dfcf1654f37dc17cf3b80e/pkg/cri/resource-manager/resource-manager.go#L163) does? E.g. call `m.control.StartStopControllers()`

Reply:
Yes, you're right. I'll fix that.

Reply:
Fixed.